Source/Samples/100_HelloSystemUI/HelloSystemUI.h

```diff
@@ -23,6 +23,8 @@
 #pragma once
 
 #include "Sample.h"
+#include "Urho3D/SystemUI/Gizmo.h"
+
 #include <Urho3D/SystemUI/SystemMessageBox.h>
```

Comment:
Use same include style for engine headers and new line ought to be between `Sample.h` and `Gizmo.h`